### PR TITLE
Docs updates.

### PR DESCRIPTION
**Pull request by Harwayne**
Docs updates.

### CODE REVIEW

File: docs/install/README.md [hunk 1]
```diff
@@ -10,13 +10,16 @@ ko apply -f ./config
...
+- Install new CRDs:
+  - `channels.pubsub.cloud.run`
   - `pullsubscriptions.pubsub.cloud.run`
-- install a controller to operate on the new resources,
+  - `topics.pubsub.cloud.run`
+- Install a controller Deployment to operate on the new resources,
   `cloud-run-events/cloud-run-events-controller`.
-- install a new service account, `cloud-run-events/cloud-run-events-controller`.
-- provide RBAC for that service account.
+- Install a webhook Deployment to validate and mutate the new resources, `cloud-run-events/cloud-run-events-webhook`.
```
Comment (Harwayne) [inline docs/install/README.md]: Note that this is actually incorrect, the Deployments are named `webhook` and `controller`. I think we should change the names of the Deployments to match these docs. Thougths?

File: docs/install/README.md [hunk 2]
```diff
@@ -10,13 +10,16 @@ ko apply -f ./config
 
 Applying the `config` dir will:
 
-- create a new namespace, `cloud-run-events`.
-- install new CRDs,
+- Create a new namespace, `cloud-run-events`.
+- Install new CRDs:
+  - `channels.pubsub.cloud.run`
```
Comment (n3wscott) [inline docs/install/README.md]: not yet for channels. It will not be here.

File: docs/install/README.md (hunk 1 above)
Comment (n3wscott) [reply docs/install/README.md]: I want them to be called `webhook` and `controller`. Can you update the doc?

File: docs/pullsubscription/README.md [hunk 3]
```diff
@@ -68,44 +72,47 @@ Pub/Sub topic, such as
...
+           ```shell
+           kubectl apply --filename pullsubscription.yaml
+           ```
+      1. Running outside Google Kubenetes Engine or want to use a Pub/Sub Topic from a different
+         project than your GKE cluster:
 
-   If you're in the samples directory, you can replace `MY_PROJECT` and
-   `TOPIC_NAME` and then apply in one command:
+           If you're in the samples directory, you can replace `MY_PROJECT`,
+           `TOPIC_NAME`, uncomment `#secret:`, and then apply in a single command:
```
Comment (n3wscott) [inline docs/pullsubscription/README.md]: secret has a default value and is not required it you follow this guide.

File: docs/pullsubscription/README.md [hunk 4]
```diff
@@ -191,7 +198,7 @@ If you are replaced `TOPIC_NAME` manually, then make sure you delete the
 resulting YAML:
```
Comment (n3wscott) [inline docs/pullsubscription/README.md]: This sentence originally does not make sense. maybe 

`then make sure you delete the  resulting YAML`

becomes

`then you can use the updated files to delete using:`

File: docs/pullsubscription/README.md [hunk 5]
```diff
@@ -12,10 +12,14 @@ Pub/Sub topic, such as
 1. Create a
    [Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
    and install the `gcloud` CLI and run `gcloud auth login`. This sample will
-   use a mix of `gcloud` and `kubectl` commands. The rest of the sample assumes
```
Comment (n3wscott) [inline docs/pullsubscription/README.md]: this language came from the docs writers

File: docs/pullsubscription/README.md [hunk 6]
```diff
@@ -25,7 +29,7 @@ Pub/Sub topic, such as
    gcloud services enable pubsub.googleapis.com
    ```
 
-1. Create a
+1. [TODO - This is only required outside of GKE] Create a
```
Comment (n3wscott) [inline docs/pullsubscription/README.md]: This is required for GKE too

File: docs/pullsubscription/README.md [hunk 7]
```diff
@@ -68,44 +72,47 @@ Pub/Sub topic, such as
    gcloud pubsub topics create $TOPIC_NAME
    ```
 
-1. Update `TOPIC_NAME` in the [`pullsubscription.yaml`](pullsubscription.yaml)
-   and apply it.
+1. Replace the placeholders with real values.
```
Comment (n3wscott) [inline docs/pullsubscription/README.md]: again, this language was taken from the docs writers.

File: docs/pullsubscription/README.md [hunk 8]
```diff
@@ -68,44 +72,47 @@ Pub/Sub topic, such as
...
 
-   If you're in the samples directory, you can replace `TOPIC_NAME` and apply in
-   one command:
+      1. Placeholders:
+          - `TOPIC_NAME` - Must be replaced.
+          - `MY_PROJECT` - Replaced only if you are running outside of Google Kubernetes Engine.
```
Comment (n3wscott) [inline docs/pullsubscription/README.md]: Or if you are targeting a topic from another project.

File: docs/pullsubscription/README.md [hunk 9]
```diff
@@ -68,44 +72,47 @@ Pub/Sub topic, such as
...
+            Note that if you are replacing this, then you should also uncomment the `#secret:` line.
```
Comment (n3wscott) [inline docs/pullsubscription/README.md]: This is not true if the secret is from the other project.

File: docs/pullsubscription/README.md [hunk 10]
```diff
@@ -68,44 +72,47 @@ Pub/Sub topic, such as
    gcloud pubsub topics create $TOPIC_NAME
    ```
 
-1. Update `TOPIC_NAME` in the [`pullsubscription.yaml`](pullsubscription.yaml)
-   and apply it.
+1. Replace the placeholders with real values.
 
-   If you're in the samples directory, you can replace `TOPIC_NAME` and apply in
-   one command:
+      1. Placeholders:
```
Comment (n3wscott) [inline docs/pullsubscription/README.md]: I don't understand why this is now a substep, there are only a few steps, seems to make sense to just have steps 1-4.

**Comment by n3wscott** [comment]
/close 

out of date says Adam.

**Pull request closed without merging**